src/client/datascience/jupyter/kernels/cellExecution.ts

```diff
@@ -622,16 +638,9 @@ export class CellExecution {
         }
     }
 
-    private async handleExecuteInput(
-        msg: KernelMessage.IExecuteInputMsg,
-        _clearState: RefBool,
-        loggers: INotebookExecutionLogger[]
-    ) {
+    private async handleExecuteInput(msg: KernelMessage.IExecuteInputMsg, _clearState: RefBool) {
```

Comment:
ExecuteInput reply is logged when Jupyter starts executing some code. It's the appropriate place to change execution count, but not the appropriate place to log a postExecute so I moved the logging up to the correct location.
https://jupyter-client.readthedocs.io/en/stable/messaging.html#code-inputs

Reply:
Also removed the hardcoded slient here. Both of these issues might have been causing problems for anything looking at the loggers for use with native notebooks.